recipes/cpython/all/patches/3.10/3.10.14-0002-disable-nis.patch

```diff
@@ -0,0 +1,11 @@
+--- a/setup.py
++++ b/setup.py
+@@ -2594,7 +2594,7 @@
+             ))
+ 
+     def detect_nis(self):
+-        if MS_WINDOWS or CYGWIN or HOST_PLATFORM == 'qnx6':
++        if True:
```

Comment:
I'm guessing this is to prevent all installations from detecting nis, instead of having some builds with and some without?

Reply:
Yes, exactly.